### PR TITLE
Move gamma, zeta calc consts to sc.yaml

### LINKED ISSUE #1035: Move block reward constants to sc.yaml
Currently in block_reward, we use various constants for zeta, gamma calculation used to compute reward shares based on other variables. Once a user-friendly name is finalized, we should move these constants to sc.yaml in [`block_reward` section](https://github.com/0chain/0chain/blob/3fea102ed86641d30f5dcdf009b3c05ff0b1eab2/docker.local/config/sc.yaml#L150)
```
const (
		// constants for gamma
		alpha = 1
		A     = 1
		B     = 1

		// constants for zeta
		I  = 1
		K  = 1
		mu = 1
	)
```

### PR DESCRIPTION
**Pull request by lawRathod**
## Fixes
closes #1035

## Changes
Moves various consts for gamma, zeta calculations to sc.yaml config.

## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/config.go [hunk 1]
```diff
@@ -45,14 +45,28 @@ type writePoolConfig struct {
...
+	QualifyingStake         currency.Coin    `json:"qualifying_stake"`
+	SharderWeight           float64          `json:"sharder_weight"`
+	MinerWeight             float64          `json:"miner_weight"`
+	BlobberWeight           float64          `json:"blobber_weight"`
+	TriggerPeriod           int64            `json:"trigger_period"`
+	Gamma                   blockRewardGamma `json:"gamma"`
+	Zeta                    blockRewardZeta  `json:"zeta"`
+}
+
+type blockRewardGamma struct {
```
Comment (dabasov) [inline code/go/0chain.net/smartcontract/storagesc/config.go:60]: add please  validation logic for new fields

**Review by dabasov — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/smartcontract/storagesc/config.go (hunk 1 above)
Comment (peterlimg) [reply code/go/0chain.net/smartcontract/storagesc/config.go:60]: @sum-it08 can you check what validation should be applied for the Gama params

Comment (ssardana08) [reply code/go/0chain.net/smartcontract/storagesc/config.go:60]: a basic validation to all should be that these constants > 0
all 6 values of `blockRewardGamma` and `blockRewardZeta`

File: docker.local/config/sc.yaml [hunk 2]
```diff
@@ -202,6 +202,14 @@ smart_contracts:
       trigger_period: 30
       blobber_capacity_ratio: 10
       blobber_usage_ratio: 40
+      gamma:
```
Comment (ssardana08) [inline docker.local/config/sc.yaml:203]: change the values to following:
A: 10
B: 9
alpha: 0.2
i: 1
K: 0.9
mu: 0.2

we need to get these changes in helm as well.

**Review by ssardana08 — DISMISSED** [review]

File: code/go/0chain.net/smartcontract/storagesc/config_settigns.go [hunk 3]
```diff
@@ -835,12 +895,12 @@ func (ssc *StorageSmartContract) commitSettingChanges(
 	}
 
 	if err = conf.validate(); err != nil {
-		return "", common.NewError("update_settings", err.Error())
+		return "", common.NewError("update_settings_validate", err.Error())
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/config_settigns.go:898]: @lawRathod please also check if those gamma values is > 0 in `conf.validate()`

**Review by peterlimg — DISMISSED** [review]

File: code/go/0chain.net/smartcontract/storagesc/config.go (hunk 1 above)
Comment (lawRathod) [reply code/go/0chain.net/smartcontract/storagesc/config.go:60]: Done!

File: docker.local/config/sc.yaml (hunk 2 above)
Comment (lawRathod) [reply docker.local/config/sc.yaml:203]: Done

File: code/go/0chain.net/smartcontract/storagesc/config_settigns.go (hunk 3 above)
Comment (lawRathod) [reply code/go/0chain.net/smartcontract/storagesc/config_settigns.go:898]: Done

**Comment by peterlimg** [comment]
lgtm, please fix system test with @Kishan-Dhakan

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**